### ISSUE TITLE
bundle lock should accept most of the options of bundle install

### ISSUE THREAD

**Issue by axos88** [bundler/bundler#6735]
Such as --gemfile and also --binstubs, --force, --with, --without

**Comment by segiddins** [comment]
The only option that would really make sense for `bundle lock` would be `--gemfile`, the other options you mention only really effect installation

**Comment by axos88** [comment]
Yeah, you're probably right, it was the one i was missing actually.

**Comment by colby-swandale** [comment]
Anyone that's looking to implement this, checkout https://github.com/bundler/bundler/pull/6676
